projects/kit/components/input-tag/input-tag.style.less

```diff
@@ -162,7 +162,7 @@
         white-space: nowrap;
     }
 
-    :host:not(:focus-within) &:last-of-type:after {
+    :host:not(._hosted-dropdown-opened) &:last-of-type:after {
```

Comment:
You can probably just use `:not(._focused)` which comes from `Interactive` and not introduce a new getter.

Reply:
fixed